testinstance/testinstance.go

```diff
@@ -112,7 +113,11 @@ func NewInstance(ctx context.Context, net tn.Network, p tnet.Identity, netOption
 	bsdelay := delay.Fixed(0)
 
 	adapter := net.Adapter(p, netOptions...)
-	dstore := ds_sync.MutexWrap(delayed.New(ds.NewMapDatastore(), bsdelay))
+	st, err := ds.NewMapDatastore(key.KeyTypeBytes)
```

Comment:
st naming is confusing, mapds is better

Reply:
also key.KeyTypeBytes chaged the behavior of original code (equivalent to StrKey), should use KeyTypeString?